src/Centreon/Application/Controller/Administration/ParametersController.php

```diff
@@ -80,13 +84,17 @@ public function getParameters(): View
         $refreshInterval = '';
         $isAcknowledgementPersistent = false;
         $isAcknowledgementSticky = false;
+        $isDowntimeFixed = false;
```

Comment:
I think it should be more relevant if a downtime is fixed by default

Reply:
Done